src/v2/view-builder/views/EnrollProfileUpdateView.js

```diff
@@ -20,7 +20,7 @@ const Body = BaseForm.extend({
     uiSchemas.forEach(input => {
       if (input.name === 'userProfile.secondEmail') {
         input.explain = View.extend({
-          template: hbs`{{i18n code="oie.profile.additional.secondemail.subtitle" bundle="login" $1="<strong>$1</strong>"}}`
+          template: hbs`{{{i18n code="oie.profile.additional.secondemail.subtitle" bundle="login" $1="<span class='strong'>$1</span>"}}}`
```

Comment:
@jmelberg-okta this works.. this was also in front of my eyes 😬  .. basically `<strong></strong>` is not working as expected..